src/main/app.ts

```diff
@@ -62,17 +66,17 @@ app.use((req, res, next) => {
 const errorLogger = new ErrorLogger()
 app.use((err, req, res, next) => {
   errorLogger.log(err)
-  const view = (env === 'mocha' || env === 'development' || env === 'dev' || env === 'demo') ? 'error_dev' : 'error'
   res.status(err.statusCode || 500)
-  res.render(view, {
-    error: err,
-    title: 'error'
-  })
-  next()
-})
-
-const accessLogger = new AccessLogger()
-app.use((req, res, next) => {
-  res.on('finish', () => accessLogger.log(req, res))
+  if (err.statusCode === 302 && err.associatedView) {
```

Comment:
I am not sure why you have added 302 status code here? I am not sure we will be moving resources temporarily. any scenario?